src/routes/index.jsx

```diff
@@ -187,15 +187,23 @@ export default function Home() {
           {/* Chat */}
           <ContentSection id="chat" title="Chat">
             <div class="flex flex-col items-center gap-3 w-full max-w-md">
-              <p class="text-center">Join our community chat to connect with other attendees!</p>
-              <p class="text-center">For everything NixCon-related and announcements there is a Matrix room you can join at #nixcon:matrix.org</p>
+              <p class="text-center">Join our community chat to connect with other attendees! The Matrix room is the place to go for announcements and everything NixCon-related.</p>
               <a
                 href="https://matrix.to/#/#nixcon:matrix.org"
                 target="_blank"
                 rel="noopener noreferrer"
                 class="px-4 py-2 bg-white/10 hover:bg-white/20 transition-colors rounded-lg text-white"
               >
-                Join Matrix Room
+                Join Matrix room
+              </a>
+              <p class="text-center mt-4">If you want to support the event as a helping hand on site, we are happy to hear from you in the dedicated Matrix orga room. Thanks!</p>
+              <a
+                href="https://matrix.to/#/#nixcon2025-orga:lassul.us"
+                target="_blank"
+                rel="noopener noreferrer"
+                class="px-4 py-2 bg-white/10 hover:bg-white/20 transition-colors rounded-lg text-white"
+              >
+                Join Matrix orga room
```

Comment:
The room is not about how to organize Matrix, but about organizing NixCon. Thus:
```suggestion
                Join orga Matrix room
```
or
```suggestion
                Join Matrix room #nixcon2025-orga
```
or
```suggestion
                Join Matrix room #nixcon2025-orga:lassul.us
```
or
```suggestion
                Join Matrix room “NixCon 2025 orga”
```
(I think I prefer the last option.)

(Maybe mention the other room's name in the other button, too.)

Reply:
Don't you think this is unnecessarily nitpicky?

The buttons would be quite large then without adding information that is not presented in the paragraphs.

I'm willing to go forward with this PR, but let me know what you think, so I can close it without wasting my time, thanks.

Reply:
> Don't you think this is unnecessarily nitpicky?

Probably. :thinking: "Matrix orga room" **did** seem wrong to me yesterday, as I had parsed it as "Join ((Matrix orga) room)" yesterday, though today I've realized that it can **also** be parsed as "Join (Matrix (orga room))" which gives the intended meaning. While some might parse it wrongly like I initially did, given the context the intended meaning is probably clear enough regardless.

So a good option is to merge this PR _as-is_ (as it undoubtedly **is** an improvement over mentioning only one room for both purposes) and (me) to follow up with a new issue or PR to improve it further. I think I might be able to come up with a better change for that than my suggestions above.